File: Java/Teste/src/Teste.java
package teste;
public class Teste {
    public static void main(String[] args) throws Exception {
        Caneta caneta1 = new Caneta("Cis Spiro", "Azul e Branco", 0.7f);
        caneta1.destampar();
        caneta1.rabiscar();
        caneta1.tampar();
        caneta1.status();
    }
}

File: Testes/C#/Teste.cs
using System;

namespace Teste {
    class Program {
        static void Main(string[] args) {
            Caneta caneta1 = new Caneta("Cis", "Azul e Branco", 0.7f);
            caneta1.Destampar();
            caneta1.Rabiscar();
            caneta1.Tampar();
            caneta1.Status();
        }
    }
}